var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -167,9 +168,10 @@ def configure_args(self):
 
         if '+mpi' in self.spec:
             options += [
-                'CC={0}'.format(spec['mpi'].mpicc),
-                'CXX={0}'.format(spec['mpi'].mpicxx),
-                'FC={0}'.format(spec['mpi'].mpifc),
+                'CC=mpiicc',
+                'CXX=mpiicpc',
+                'FC=mpiifort',
+                'FCFLAGS=-mkl=cluster',
```

Comment:
NACK

Reply:
Yes, I am the person in charge of compiling and installing this software on the HPC cluster. For the best effect, I only test the Intel compiler